tests/ext/extension_no_static_tls.phpt

```diff
@@ -0,0 +1,44 @@
+--TEST--
+Extension is not compiled with STATIC_TLS
```

Comment:
It would be great if you could add a similar test to appsec (in this repository, not the old one).